SlevomatCodingStandard/Helpers/NamespaceHelper.php

```diff
@@ -29,7 +30,14 @@ class NamespaceHelper
 	 */
 	public static function getAllNamespacesPointers(File $phpcsFile): array
 	{
-		return TokenHelper::findNextAll($phpcsFile, T_NAMESPACE, 0);
+		static $cache = [];
+
+		$cacheKey = $phpcsFile->getFilename() . $phpcsFile->getFixedCount();
```

Comment:
Please see https://github.com/slevomat/coding-standard/blob/master/SlevomatCodingStandard/Helpers/UseStatementHelper.php#L123 and implement it likewise.

Reply:
I moved this logic to SniffLocalCache helper, what do you think about it?